package.json

```diff
@@ -48,6 +48,7 @@
     "fileuploader": "^1.2.7",
     "firebase": "^7.20.0",
     "hammerjs": "^2.0.8",
+    "moment": "^2.29.1",
```

Comment:
we don't need to set the dependency for 1 use only. so please consider using native js.

Reply:
we already have moment.js, but older version. I've just updated the existing one. Here is the screen from dev branch:
![image](https://user-images.githubusercontent.com/46484914/116042380-00393400-a677-11eb-9fcd-84751ccfd239.png)

Reply:
this screen from package-lock.json?

Reply:
Yes, indeed